rpc/client_server_test.go

```diff
@@ -50,6 +51,13 @@ func (d *dummyRPCHandler) SubscribeToOrders(ctx context.Context) (*rpc.Subscript
 	return d.subscribeToOrdersHandler(ctx)
 }
 
+func (d *dummyRPCHandler) SubscribeToHeartbeat(ctx context.Context) (*rpc.Subscription, error) {
```

Comment:
We don't need a `SubscribeToHeartbeat` method on `dummyRPCHandler` anymore since it is not part of the `RPCHandler`.